src/serviceFacades/notifications.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @author flynn
+ *
+ */
+import callApi from "../common/callApi";
+
+const NOTIFICATIONS_LAST_TEN_KEY = "fetchNotificatons";
```

Comment:
This variable has been renamed, which is good, but I think Paul also intended for this string to be more descriptively named.  The string serves as a key for `react-query` to recognize different queries being run and `fetchNotifications` could accidentally be used as a key for getting all notifications instead of just the last 10, for example